src/components/Header/Header.js

```diff
@@ -1,10 +1,13 @@
 import React, { useMemo } from 'react'
```

Comment:
I'm not sure what I should try to reproduce here. Is the intention to alway get to the particular Garden's home? It seems it always take me to the Dashboard.

Reply:
Yes, if you are in a specific garden then clicking on the logo should direct you to that garden's home.

Before it would take you to the gardens list view.